packages/http/src/mocker/__tests__/HttpMocker.spec.ts

```diff
@@ -676,5 +689,31 @@ describe('mocker', () => {
         });
       });
     });
+
+    describe('should return 500 error', () => {
```

Comment:
This test does not seem relevant to the PR that has been opened. Could you please add a test that covers the original circular reference problem and show that it is no longer throwing a 500?